vpd-tool/include/tool_json_utility.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <nlohmann/json.hpp>
+
+#include <fstream>
+
+namespace vpd
+{
+namespace jsonUtility
+{
+
+/**
+ * @brief API to parse respective JSON.
+ *
+ * Exception is thrown in case of JSON parse error.
+ *
+ * @param[in] i_pathToJson - Path to JSON.
+ * @return Parsed JSON.
+ * @throw std::runtime_error
+ */
+inline nlohmann::json getParsedJson(const std::string& i_pathToJson)
+{
+    if (i_pathToJson.empty())
+    {
+        throw std::runtime_error("Path to JSON is missing");
+    }
+
+    if (!std::filesystem::exists(i_pathToJson) ||
```

Comment:
Please handle exception for this API.

Reply:
Added exception handling here. This `getParsedJson` API implementation is copied from `utility/json_utility.hpp` . This exception is not handled there.